Dockerfile

```diff
@@ -20,7 +20,8 @@ WORKDIR $GHOST_INSTALL
 #Install required packages
 RUN set -eux                            && \
     apk update && apk add python make   && \
-    chown node:node "$GHOST_INSTALL"     
+    chown node:node "$GHOST_INSTALL"    && \
```

Comment:
The Ghost Install Folder is the one inside the container (where we are installing files) and not the one we are mounting.
It seems the problem you are having are related to your external folder... but for this one nothing has changed: the run user was node and is still node.
So I think changing stuffs here will not allow you to fix the problem

Reply:
Ok, it might be something else in my setup. Need to test things more deeply.
Thanks!

Reply:
I still need to do the chown since it breaks OpenShift/Kubernetes which always fallbacks to ``node`` @pascalandy